src/components/FrameMetadata.test.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import { FrameMetadata } from './FrameMetadata';
+
+describe('FrameMetadata', () => {
+  it('renders', () => {
+    expect(FrameMetadata).toBeDefined();
+  });
+
+  it('renders with image', () => {
+    const meta = render(<FrameMetadata image="https://example.com/image.png" />);
+    expect(meta.container.querySelector('meta[property="fc:frame:image"]')).not.toBeNull();
```

Comment:
nit: the not null check is redundant when we also check that the value is defined in the next expect

Reply:
Nice